### ISSUE TITLE
external page doesn't seem to download

### ISSUE THREAD

**Issue by takase1121** [calvinaquino/LNReader-Android#290]
The external page doesn't seem to download. There is no toast showing that its downloaded. At first I thought the toast was disabled but looking at the code give me idea that the toast wasn't disabled. Is it a bug?

**Comment by Nandaka** [comment]
nope, it does load in my device. Can you try to use different device and check? You are using custom rom, right?

**Comment by Nandaka** [comment]
From discord.

@Nandaka Sorry for the ping, but I just wanted to share a discovery with you.

As I'm sure you've heard over time, several people have reported issues updating content over Wi-Fi, yet have no trouble over mobile data. I was one of those people. In fact, other apps (like Manga Rock) behaved in a similar fashion for me.

Poking around Android help with similar issues, someone mentioned disabling IPv6 at the router. With mine specifically, I disabled IPv6 - Automatic, and now the app updates flawlessly, (and the other apps load perfectly fine as well).
